NUG-new/pages/nug_conventions.md

```diff
@@ -0,0 +1,13 @@
+---
+title: NUG Conventions
+last_updated: 2021-04-06
+sidebar: nnug_sidebar
+toc: false
+permalink: nug_convensions.html
```

Comment:
Typo, should permalink be `nug_conventions.html` ?

Reply:
Thanks @WardF for catching that. Just pushed fix.